Source/Blazorise/Components/Droppable/DropZone.razor.cs

```diff
@@ -504,6 +510,11 @@ protected override bool ShouldRender()
     /// If true, the reordering of the items will be enabled.
     /// </summary>
     [Parameter] public bool AllowReorder { get; set; }
+    
+    /// <summary>
+    /// The callback that is raised when the order of items changed. Only if <see cref="AllowReorder"/> is enabled.
+    /// </summary>
+    [Parameter] public Func<Dictionary<TItem, int>, Task> Reordered { get; set; }
```

Comment:
Now that I think of it might be better to have a "context" object instead of `Dictionary<TItem, int>`.

```cs
class DropDownOrderContext<TItem>
{
  public TItem Item {get;}
  public int Order {get;} // or Index
}
```

This way the API is more flexible for any change in the future.

Reply:
And then passing as a List?

What about 

```csharp
public class DropzoneOrderContext<TItem>
{
   
    public Dictionary<TItem, int> Indeces{ get; set; }

}
```

or even:

```csharp
public class DropzoneOrderContext<TItem>
{

    public List<DropzoneOrderedItem> OrderedItems { get; set; }
   //something else when needed.

}

public class DropzoneOrderedItem<TItem>
{
 public TItem Item {get;}
  public int Order {get;} 
}
```

Reply:
I like the second example.

Reply:
PS. Just match the naming accordingly to component name. `DropZoneOrderContext`, and `DropZoneOrderedItem`.